SSA/Experimental/Bits/Fast/BitStream.lean

```diff
@@ -431,11 +478,8 @@ theorem neg_congr (e1 : a ≈ʷ b) : (-a) ≈ʷ -b := by
     simp only [ih (by omega), Bool.bne_right_inj, and_self]
   simp only [Neg.neg, BitStream.neg, neg_congr_lemma]
 
-theorem not_congr (e1 : a ≈ʷ b) : (~~~a) ≈ʷ ~~~b := by
-  intros g h
-  simp only [not_eq, e1 g h]
 
-theorem equal_trans  (e1 :  a ≈ʷ b) (e2 : c ≈ʷ d)  : (a ≈ʷ c) = (b ≈ʷ d) := by
+theorem equal_trans  (e1 : a ≈ʷ b) (e2 : c ≈ʷ d) : (a ≈ʷ c) = (b ≈ʷ d) := by
```

Comment:
I dont quite understand how this is a `trans` theorem? it feels more like theres some diagram youre talking about commutativity in